setup.py

```diff
@@ -153,6 +153,7 @@ def read(name):
         'humanize',
         'tqdm',
         'posthog',
+        'distro',
```

Comment:
I'm not fond of adding new dependencies. I actually want to remove some of them. I saw we only use this to determine the linux version. Do you think having that level of detail is important?

Reply:
Usually when bugs happens, it's easier to recreate within the exact same env (and version). I don't think it's that crucial, let's discuss the tradeoffs.

Reply:
I think let's keep if for now after we saw it's not heavy.